acceptance/runUIAcceptance.sh

```diff
@@ -107,6 +109,8 @@ fi
 HOSTNAME=`hostname -s`
 HOST_IP=`hostname -i`
 
+trap 'docker rm -f ui_acceptance' INT
```

Comment:
It closes the docker container if you ctrl c out of the test suite. Before, if the test suite didn't end normally, you would have to manually kill and remove the docker container.